packages/components/src/internal/components/picklist/actions.ts

```diff
@@ -213,8 +195,8 @@ export function getSelectedPicklistSamples(
     queryParameters?: Record<string, any>,
     viewName?: string
 ): Promise<number[]> {
-    return new Promise((resolve, reject) => {
-        getOrderedSelectedMappedKeys(
+    try {
```

Comment:
No reason to try/catch here as `getOrderedSelectedMappedKeys()` and `setSnapshotSelections()` will already log the error to the console.